ios/RNDatePicker/RNDatePickerManager.m

```diff
@@ -114,6 +114,15 @@ - (UIView *)view
             [picker setTimeZone:[RCTConvert NSTimeZone:timeZoneProp]];
         }
         
+        NSString * _Nonnull theme = [RCTConvert NSString:[props objectForKey:@"theme"]];
+        if ([theme isEqualToString:@"light"]) {
+            alertController.overrideUserInterfaceStyle = UIUserInterfaceStyleLight;
+        } else if ([theme isEqualToString:@"dark"]) {
+            alertController.overrideUserInterfaceStyle = UIUserInterfaceStyleDark;
+        } else {
+            alertController.overrideUserInterfaceStyle = UIUserInterfaceStyleUnspecified;
+        }
+        
```

Comment:
Can you guard these conditionals as done in DatePicker.m? Then it will still compile on ios 12.

Reply:
Done!